iroh-node-util/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "iroh-node-util"
+license = "MIT OR Apache-2.0"
+version = "0.1.0"
+edition = "2021"
+authors = ["n0 team"]
+repository = "https://github.com/n0-computer/iroh"
+keywords = ["quic", "networking", "holepunching", "p2p"]
+
```

Comment:
missing some fields we use on other cargo.tomls

Reply:
added description and readme. not sure what else to add.